lms/djangoapps/courseware/tests/test_microsites.py

```diff
@@ -142,6 +142,26 @@ def test_not_microsite_anonymous_homepage_content(self):
         # assert that footer template has been properly overriden on homepage
         self.assertNotContains(resp, 'This is a Test Microsite footer')
 
+    @override_settings(SITE_NAME=settings.MICROSITE_TEST_HOSTNAME)
+    def test_microsite_anonymous_copyright_content(self):
+        """
+        Verify that the copyright, when accessed via a Microsite domain, returns
+        HTML that reflects the Microsite branding elements
```

Comment:
We're not really validating any particular Microsite branding elements or HTML in this test -- we're simply confirming that the correct page content is being returned.  If there are specific items we should be looking for, we should add them to the test, otherwise we should revise the docstring to describe what is being verified here.